ts_sdk/task/__util_log.py

```diff
@@ -30,26 +30,24 @@ def log(self, *args):
                 __builtins__["default_print"](
                     json.dumps(self.create_json_log_entry(arg))
                 )
-            except TypeError:
-                __builtins__["default_print"](
-                    TypeError("Object of type type is not JSON serializable")
-                )
+            except Exception as ex:
+                __builtins__["default_print"](TypeError(repr(ex)))
```

Comment:
Since this catches any exception, it isn't necessarily a `TypeError`.  I'd suggest going with:
```suggestion
            except TypeError as ex:
                __builtins__["default_print"](repr(ex))
```